Framework/Kernel/inc/MantidKernel/AttenuationProfile.h

```diff
@@ -17,10 +17,14 @@ class Material;
 
 class MANTID_KERNEL_DLL AttenuationProfile {
 public:
+  AttenuationProfile(){};
```

Comment:
Doesn't look like this empty constructor is required?

Reply:
It is used in initialising an attenuation profile for the tests